integrated_channels/moodle/client.py

```diff
@@ -337,7 +336,9 @@ def create_course_completion(self, user_id, payload):  # pylint: disable=unused-
             'grades[0][grade]': completion_data['grade'] * 100
         }
 
-        return self._post(params)
+        # The base learner data transmitter expects a (code, body) tuple to be returned
```

Comment:
it would be good to see if there is a test that captures this.. any potential to do that?

Reply:
Uhh, yeah! Can do